src/app/Http/Controllers/Operations/DeleteOperation.php

```diff
@@ -29,11 +30,7 @@ protected function setupDeleteDefaults()
     {
         $this->crud->allowAccess('delete');
 
-        $this->crud->operation('delete', function () {
-            $this->crud->loadDefaultOperationSettingsFromConfig();
-        });
```

Comment:
Would it really be that terrible to need to do this if you want a config file for the operation?

```php
        LifecycleHook::hookInto('delete:before_setup', function () {
            $this->crud->loadDefaultOperationSettingsFromConfig();
        });
```